src/geometry/manifold/instances/real.lean

```diff
@@ -82,59 +82,22 @@ a model for manifolds with boundary. In the locale `manifold`, use the shortcut
 def model_with_corners_euclidean_half_space (n : ℕ) [has_zero (fin n)] :
   model_with_corners ℝ (euclidean_space ℝ (fin n)) (euclidean_half_space n) :=
 { to_fun      := λx, x.val,
```

Comment:
could you also update this one to use the coercion?

Reply:
```suggestion
{ to_fun      := coe,
```